doc/classes/Environment.xml

```diff
@@ -82,6 +82,54 @@
 		<member name="background_mode" type="int" setter="set_background" getter="get_background" enum="Environment.BGMode" default="0">
 			The background mode. See [enum BGMode] for possible values.
 		</member>
+		<member name="dynamic_gi_bounce_feedback" type="float" setter="set_dynamic_gi_bounce_feedback" getter="get_dynamic_gi_bounce_feedback" default="1.0">
+			How much light bounces back to the probes. This increases the amount of indirect light received on surfaces.
```

Comment:
This still seems to be the case as of the latest revision of this PR, so I suggest documenting it:

```suggestion
			How much light bounces back to the probes. This increases the amount of indirect light received on surfaces.
			[b]Note:[/b] Values higher than [code]1.0[/code] may result in infinite feedback loops with bright surfaces. This can cause GI to appear extremely bright over time.
```